src/scss/_common-functions.scss

```diff
@@ -9,4 +9,14 @@
 
 @function z($key) {
   @return _get-map-prop($z-layers, $key);
+}
+
+@function rems2dp($size) {
```

Comment:
I doubt it, rem is dependant on font size, we probably want it to be more a fixed size on native